src/FakeTime/FakeTimeTest.ino

```diff
@@ -53,6 +53,58 @@ test(should_handle_seconds_errors_correctly) {
   assertEqual(formattedTimeBuffer, "<initial value>");
 }
 
+test(should_correctly_add_second_and_a_half) {
+  // given
+  char formattedTimeBuffer[20] = "<initial value>";
+  FakeTime tested = FakeTime(0, 1, 2);
+
+  // when
+  tested.plusMillis(1500);
+  tested.formatTime(formattedTimeBuffer);
+
+  // then
+  assertEqual(formattedTimeBuffer, "00:01:03");
+}
+
+test(should_correctly_add_twelve_seconds) {
+  // given
+  char formattedTimeBuffer[20] = "<initial value>";
+  FakeTime tested = FakeTime(17, 00, 00);
+
+  // when
+  tested.plusMillis(12000);
+  tested.formatTime(formattedTimeBuffer);
+
+  // then
+  assertEqual(formattedTimeBuffer, "17:00:12");
+}
+
+test(should_correctly_add_five_minutes_and_33_seconds) {
```

Comment:
Nice. How do you count that?

Reply:
To calculate seconds:

```c
long seconds = millis / 1000;
```

then from the number of seconds we need to calculate *minutes* part and *seconds* part:

```c
long seconds = (millis / 1000) % 60;
long minutes = seconds / 60;
```

and we need to somehow extend this approach for hours...

Reply:
maybe
`long hoursToAdd = minutesToAdd / 60;`